caffe2/operators/onnx_while_op.cc

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "caffe2/operators/onnx_while_op.h"
+
+namespace caffe2 {
+
+REGISTER_CPU_OPERATOR(ONNXWhile, ONNXWhileOp<CPUContext>);
```

Comment:
Is there a reason why we're putting `ONNX` in the name here? Could we just go with `While` instead?

Reply:
This is more of a working name to distinguish this from the existing `While` operator. I wanted to get this in code to make sure the operator semantics for the ONNX spec were sane from a backend point of view. We probably should merge this with the existing operator at some point